src/repositories/PostRepository.ts

```diff
@@ -7,24 +8,27 @@ import { Uuid } from '../types';
 @EntityRepository(PostModel)
 export class PostRepository extends AbstractRepository<PostModel> {
   public async getAllPosts(): Promise<PostModel[]> {
-    return await this.repository.createQueryBuilder("post")
-    .leftJoinAndSelect("post.user", "user")
-    .getMany();
+    return await this.repository
+      .createQueryBuilder("post")
+      .leftJoinAndSelect("post.user", "user")
+      .where("post.archive = false")
+      .getMany();
   }
 
   public async getPostById(id: Uuid): Promise<PostModel | undefined> {
     return await this.repository
-    .createQueryBuilder("post")
-    .leftJoinAndSelect("post.user", "user")
-    .where("post.id = :id", { id })
-    .getOne();
+      .createQueryBuilder("post")
+      .leftJoinAndSelect("post.user", "user")
+      .where("post.id = :id", { id })
+      .getOne();
   }
 
   public async getPostsByUserId(userId: Uuid): Promise<PostModel[]> {
     return await this.repository
       .createQueryBuilder("post")
       .leftJoinAndSelect("post.user", "user")
       .where("user.id = :userId", { userId })
+      .andWhere("post.archive = false")
```

Comment:
unrelated to PR?

Reply:
this code was already merged into main. Will look into why its showing as newly pushed code